src/main/java/net/twasi/obsremotejava/model/KeyModifiers.java

```diff
@@ -0,0 +1,38 @@
+package net.twasi.obsremotejava.model;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.List;
+
+@Setter(AccessLevel.PRIVATE)
+@ToString
+@NoArgsConstructor
+public class KeyModifiers {
+    private boolean shift;
+    private boolean alt;
+    private boolean control;
+    private boolean command;
+
+    public static KeyModifiers fromTypeList(List<KeyModifierType> modifierList) {
```

Comment:
After thought, I'm not a fan of this extra layer.
Maybe we could simply add the Builder annotation to this and ask the developer an instance in https://github.com/Twasi/websocket-obs-java/pull/56/files#diff-2c09711e485dceba4478a51b66c08dc9d89c8ad2c2ced472da35b0041a4e7ac2R16

Reply:
Yeah that could indeed be better but I also like the idea of this. But I will remove this and simply add the builder.